libraries/AP_OSD/AP_OSD.h

```diff
@@ -162,10 +169,20 @@ class AP_OSD {
     AP_Int8 rc_channel;
     AP_Int8 sw_method;
 
+    AP_Int8 v_offset;
+    AP_Int8 h_offset;
+
+    AP_Int8 warn_rssi;
+    AP_Int8 warn_nsat;
+    AP_Float warn_batvolt;
+
     enum {
         OPTION_DECIMAL_PACK = 1U<<0,
+        OPTION_INVERTED_WIND = 1U<<1,
+        OPTION_INVERTED_AH_ROLL = 1U<<2,
+        OPTION_IMPERIAL_UNITS = 1U<<3,
```

Comment:
I think we'll need a separate OSD_UNITS parameter, as I'm sure we will end up with several possible units systems (eg. knots, m/s, mph, km/hr etc)

Reply:
Added parameter.